plugins/Actions/DataTable/Filter/Actions.php

```diff
@@ -50,8 +50,11 @@ public function filter($table)
 
             foreach ($dataTable->getRows() as $row) {
                 $url = $row->getMetadata('url');
+                $label = $row->getColumn('label');
                 if ($url) {
                     $row->setMetadata('segmentValue', urldecode($url));
+                } else if ($this->isPageTitleType && $label != -1) {
+                    $row->setMetadata('segmentValue', trim(urldecode($label)));
```

Comment:
Is the `-1` for `ID_SUMMARY_ROW`, maybe it should also check for the new totals row? Also not sure if it's really necessary, but should we check if `$label` is not false?

Reply:
The check is required. Otherwise the summary row will have a segmented log feature, which can't work. Will add that for totals row as well....

Reply:
Ok. No need to check for totals row, as that row will be added afterwards. Summary row might exist before.